Source/Core/Core/DSP/DSPCore.h

```diff
@@ -279,10 +279,10 @@ struct DSP_Regs
 struct DSPInitOptions
 {
   // DSP IROM blob, which is where the DSP boots from. Embedded into the DSP.
-  std::array<u16, DSP_IROM_SIZE> irom_contents;
+  std::array<u16, DSP_IROM_SIZE> irom_contents = {};
 
   // DSP DROM blob, which contains resampling coefficients.
-  std::array<u16, DSP_COEF_SIZE> coef_contents;
+  std::array<u16, DSP_COEF_SIZE> coef_contents = {};
```

Comment:
Those two seem to be outliers, other code uses `std::array{}` while this uses `std::array = {}`. I think we should be consistent, if the compiler lets us.